pkgs/by-name/cs/csharpier/package.nix

```diff
@@ -0,0 +1,19 @@
+{ buildDotnetGlobalTool, lib }:
+
+buildDotnetGlobalTool {
+  pname = "csharpier";
+  version = "0.26.4";
+  executables = "dotnet-csharpier";
+
+  nugetSha256 = "sha256-9ieSAWulfJZz0INRf95Nz5C865sM8kffK2UGZ1NZ6Z0=";
+
+  meta = with lib; {
+    description = "CSharpier is an opinionated code formatter for C#";
+    homepage = "https://csharpier.com/";
+    changelog = "https://github.com/belav/csharpier/blob/main/CHANGELOG.md";
+    license = licenses.mit;
+    platforms = platforms.unix;
```

Comment:
This can be dropped completely. Upstream doesn't clearly specify that they limit platforms anywhere, and `buildDotnetModule` (called through `buildDotnetGlobalTool`) handles the platforms for us.

https://github.com/NixOS/nixpkgs/blob/a5845db50b44e4cb5867babf1110d3dbe4f65687/pkgs/build-support/dotnet/build-dotnet-module/default.nix#L93-L96

```suggestion
```